src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -47,19 +49,22 @@
 
     private ObservableList<TurboIssue> issuesToDisplay = null;
 
-    protected HBox panelMenuBar;
-    protected Text nameText;
-    protected HBox nameBox;
-    protected HBox menuBarNameArea;
-    protected HBox menuBarCloseButton;
-    protected Label renameButton;
-    protected Label closeButton;
-    protected PanelNameTextField renameTextField;
+    public HTPanelMenuBar panelMenuBar;
+    private Text nameText;
+    public HBox nameBox;
+    public HBox menuBarNameArea;
+    public HBox menuBarCloseButton;
+    public HBox menuBarConfirmButton;
+    public HBox menuBarUndoButton;
+    public Label renameButton;
+    public Label closeButton;
```

Comment:
Can we move the related parts and initialisation into PanelMenuBar then? The new class is meant to encapsulate these, and we want to avoid having to know about them individually in FilterPanel.